.github/workflows/ci.yaml

```diff
@@ -436,6 +439,117 @@ jobs:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - name: Retrieve Cached Dependencies
+        uses: actions/cache@v4
+        id: mix-cache
+        with:
+          path: |
+            deps
+            _build/dev
+            priv/plts
+          key: ${{ runner.os }}-${{ env.OTP_VERSION }}-${{ env.ELIXIR_VERSION }}-${{ hashFiles('mix.lock') }}
+      - name: Retrieve NPM Cached Dependencies
+        uses: actions/cache@v4
+        id: npm-cache
+        with:
+          path: |
+            assets/node_modules
+          key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ hashFiles('assets/package-lock.json') }}
+      - name: Retrieve E2E NPM Cached Dependencies
+        uses: actions/cache@v4
+        id: npm-e2e-cache
+        with:
+          path: |
+            test/e2e/node_modules
+          key: ${{ runner.os }}-${{ env.NODE_VERSION }}-${{ hashFiles('test/e2e/package-lock.json') }}
+      - name: "Docker compose dependencies"
+        uses: isbang/compose-action@v1.5.1
+        with:
+          compose-file: "./docker-compose.yaml"
+          down-flags: "--volumes"
+      - name: Postgres trento_dev restore
+        uses: tj-actions/pg-restore@v6
+        with:
+          database_url: "postgresql://postgres:postgres@localhost:5433/postgres"
+          backup_file: "test/fixtures/regression/sap_system_split/trento_dev.sql"
+          postgresql_version: "15"
+      - name: Postgres trento_eventstore_dev restore
+        uses: tj-actions/pg-restore@v6
+        with:
+          database_url: "postgresql://postgres:postgres@localhost:5433/postgres"
+          backup_file: "test/fixtures/regression/sap_system_split/trento_eventstore_dev.sql"
+          postgresql_version: "15"
+      - name: Run DB migrations
+        run: mix ecto.migrate
+      - name: Run trento detached
+        run: mix phx.server &
+      - name: Install photofinish
+        uses: jaxxstorm/action-install-gh-release@v1.11.0
+        with:
+          repo: trento-project/photofinish
+          tag: v1.3.0
+          cache: enable
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Give executable permissions to photofinish
+        run: chmod +x $(whereis photofinish | cut -d" " -f2)
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        env:
+          cypress_video: false
+          cypress_db_host: postgres
+          cypress_db_port: 5432
+          cypress_photofinish_binary: $(whereis photofinish | cut -d" " -f2)
+        with:
+          working-directory: test/e2e
+          spec: |
+            cypress/e2e/databases_overview.cy.js
+            cypress/e2e/sap_systems_overview.cy.js
+          wait-on-timeout: 30
+          config: baseUrl=http://localhost:4000
+      - name: Upload cypress test screenshots
+        uses: actions/upload-artifact@v4
+        if: failure()
+        with:
+          name: e2e-screenshots
```

Comment:
Hey @rtorrero ,
Please change this `name` and `path`. 
They are a copy paste from e2e I guess. They could conflict with e2e artifcats